README.md

```diff
@@ -1,9 +1,29 @@
 The ev3dev meta repo
 ====================
-This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases),
-the ev3dev [wiki](https://github.com/ev3dev/ev3dev/wiki) and the [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
 
-To learn more about ev3dev, check out our home page at <http://www.ev3dev.org>.
+This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases) and the project-wide  [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
 
-If you are looking for code, check out all of the other repositories at <https://github.com/ev3dev>.
+### Looking for documentation and tutorials? Check out our [website](http://www.ev3dev.org).
 
+Our website hosts docs, guides and tutorials to help you get up-and-running with ev3dev as quickly as possible. Check out the [getting started guide](http://www.ev3dev.org/docs/getting-started/) if you're looking for a place to jump in.
+
+### Want to get started writing your own programs? Look through the [list of available libraries](http://www.ev3dev.org/docs/libraries/).
+
+We have many contributors that maintain libraries to make it as easy as possible to interface with ev3dev from your code. There's a [centralized repo](https://github.com/ev3dev/ev3dev-lang) with a "unified" library set as well as an overall [list of available libraries](http://www.ev3dev.org/docs/libraries/). Just pick a language, find the corresponding library, and dive into that library's examples. Or, you can always access the drivers directly if your language isn't already implemented as a pre-made library; there's  information on this style on our [docs landing page](http://www.ev3dev.org/docs/).
+
+### Searching for code? Have a look at all of the other repositories in the [ev3dev GitHub organization](https://github.com/ev3dev).
```

Comment:
We should probably make it clear that this "code" is for working on the ev3dev OS itself as opposed to example code for programming your robot.